### PR TITLE
Add number entities for Parameter.Inverter.AcALim and .WMax

### PR DESCRIPTION
**Pull request by jastBytes**
This is an PR to implement feature from issue #5. It adds two more number entities to control the maximum current and or the maximum power charged to your car.

### CODE REVIEW

**Comment by jastBytes** [comment]
Just installed it into my HA and something is now broken with all number entites. I can not figure out what's the problem right now. Will look into it soonish.

File: custom_components/smaev/number.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     NumberMode,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import EntityCategory, UnitOfEnergy, UnitOfTime
+from homeassistant.const import EntityCategory, UnitOfEnergy, UnitOfTime, UnitOfElectricCurrent
```
Comment (alengwenus) [inline custom_components/smaev/number.py:17]: `UnitOfPower` also needs to be imported

File: custom_components/smaev/number.py [hunk 2]
```diff
@@ -79,9 +79,28 @@ class SmaEvChargerNumberEntityDescription(NumberEntityDescription):
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         entity_registry_enabled_default=True,
     ),
+    SmaEvChargerNumberEntityDescription(
+        key="charge_current_limit",
+        translation_key="charge_current_limit",
+        type=SMAEV_PARAMETER,
+        channel="Parameter.Inverter.AcALim",
+        native_step=1,
```
Comment (alengwenus) [inline custom_components/smaev/number.py:87]: Is it sufficient to have steps of 1A?

File: custom_components/smaev/number.py (hunk 1 above)
Comment (jastBytes) [reply custom_components/smaev/number.py:17]: Ah, thank you! Will fix.

File: custom_components/smaev/number.py (hunk 2 above)
Comment (jastBytes) [reply custom_components/smaev/number.py:87]: IMHO that's right but I'll double check.

File: custom_components/smaev/number.py (hunk 1 above)
Comment (jastBytes) [reply custom_components/smaev/number.py]: Fixed it, tested it. Working!

File: custom_components/smaev/number.py (hunk 2 above)
Comment (jastBytes) [reply custom_components/smaev/number.py:87]: Checked that 1A steps is correct.

**Comment by alengwenus** [comment]
Thanks @jastBytes!
I changed the default visibility of the two parameters to False as I consider them as _advanced_ features (one could easily violate the regulations of the grid provider if the values are set to high). The entities can be enabled in the device settings at any time.

**Pull request merged**